articles/TOC.yml

```diff
@@ -1,33 +1,54 @@
-- name: Welcome
+- name: Quantum Development Kit Preview Documentation
   href: index.md
-- name: Getting Started
-  topicHref: install-guide/index.md
-  href: install-guide/toc.yml
-- name: Quantum computing concepts
-  topicHref: concepts/index.md
-  href: concepts/toc.yml
-- name: Quickstart - your first quantum program
-  href: quickstart.md
-- name: Managing quantum machines and drivers
-  topicHref: machines/index.md
-  href: machines/toc.yml
-- name: Quantum development techniques
-  topicHref: techniques/index.md
-  href: techniques/toc.yml
-- name: Q# libraries
-  href: libraries/toc.yml
-- name: Q# programming language
-  topicHref: language/index.md
-  href: language/toc.yml
-- name: Contributing to the Quantum Development Kit
-  href: contributing/toc.yml
-  topicHref: contributing/index.md
-- name: Glossary
-  href: Glossary.md
-- name: For more information
-  href: for-more-info.md
-- name: Release Notes
-  topicHref: relnotes/index.md
-  href: relnotes/toc.yml
-- name: License
-  href: license.md
+- name: Overview
+  items:
+  - name: Getting started
+    href: install-guide/index.md
+  - name: Contributing to the Quantum Development Kit
+    href: contributing/toc.yml
+    topicHref: contributing/index.md
+- name: Quickstarts
+  items:
+  - name: C#
+    href: install-guide/csharp.md
+  - name: Create a Bell State in Q#
+    href: quickstart.md
+  - name: Jupyter Notebooks
+    href: install-guide/jupyter.md
+  - name: Python
+    href: install-guide/python.md
+- name: Concepts
+  items:
+  - name: Quantum computing concepts
```

Comment:
The "quantum computing concepts" section is disconnected enough from Q# and the Quantum Development Kit that I worry a little bit about it being the _only_ item listed under "Concepts." Some of the content elsewhere about the Q# language and libraries might could come forward to here?

Reply:
I'm not sure. Perhaps other team members will chime in with their thoughts after the holiday weekend.  Thanks for pointing this out.

Reply:
I agree with @cgranade, probably the only item listed under concepts would be "quantum computing concepts". I didn't find anything in Q# Language that could have its subsection there. I think some parts of the "Q# standard libraries" could fit in "quantum computing concepts". 
As an idea maybe we could create a section about "Computer programming concepts" with general concepts about programming languages. For example, I come from a pure physics background and it's not rare that I need to look up in Google some terminology of the documentation related to CS concepts. I think a short section with the main things (what's a compiler, namespaces, etc) in the context of Q# could be helpful. But I don't know to what extent it is worth it.

Reply:
I looked at the Q# libraries and Q# programming concepts sections again. I think they are conceptual enough to move over to the Concepts section, and modify as necessary in subsequent doc revisions. Meanwhile, the C# library reference and C# components reference should be moved up from the bottom of the TOC to the Reference section.

Reply:
Makes sense, that sounds like a good path forward!